firefox_ios/views/firefox_ios_feature_usage_events_not_added.view.lkml

```diff
@@ -0,0 +1,545 @@
+include: "//looker-hub/firefox_ios/views/events_unnested.view.lkml"
+
+
+view: firefox_ios_feature_usage_events_not_added {
+  derived_table: {
+    sql:
+      WITH _events_ping_distinct_client_count AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS events_ping_distinct_client_count
+  FROM
+    `mozdata.firefox_ios.events_unnested`
```

Comment:
Maybe BigQuery optimized this automatically, but I think it'd be better to query `events` instead of `events_unnested` if you are just counting clients